lib/request.js

```diff
@@ -45,6 +45,14 @@ function Request(apiHost, apiKey, secret) {
     return request('PATCH', headers, that.apiHost + endpoint, contentJson);
   };
 
+  this.delete = function (endpoint, params) {
+    const contentMd5 = '';
```

Comment:
The variable is useless, I guess you can put the empty string directly in the method call.